docs/appendices/release-notes/unreleased.rst

```diff
@@ -205,3 +205,9 @@ Fixes
   consume invalid table names provided in a double-quoted string format
   containing ``.`` such as ``"table.t"`` by mis-interpreting it as
   ``"table"."t"``, which is a two double-quoted strings joined by a ``.``.
+
+- Fixed an issue that caused failure of the statements containing comparison of
+  :ref:`bit <data-type-bit>` strings with different length. An example::
+
+     CREATE TABLE tbl (b BIT(3));
+     SELECT * FROM tbl WHERE b = B'1'
```

Comment:
Maybe simplify it?
```suggestion
     SELECT B'01' = B'1'
```

Reply:
Yep, saw it in the first comment but forgot to address